save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/userprofile/UserProfileView.kt

```diff
@@ -37,14 +37,15 @@ val userProfileView: FC<UserProfileViewProps> = FC { props ->
     val (user, setUser) = useState<UserInfo?>(null)
     val (organizations, setOrganizations) = useState<List<OrganizationDto>>(emptyList())
     val (selectedMenu, setSelectedMenu) = useState(UserProfileTab.VULNERABILITIES)
+    val (vulnerabilities, setVulnerabilities) = useState<Array<VulnerabilityDto>>(emptyArray())
 
-    useRequest {
+    val initialRequests = useDeferredRequest {
```

Comment:
Why do you need to use `useDeferredRequest`? If I am not mistaken, it is used only in `useOnce`, which is the same as `useRequest { REQUEST }`

Reply:
Are you sure that `useOnce { useDeferredRequest {} }` is the same to `useRequest` ?
We have tens of places where it is used like I used it. 

I also see serious differences in implementation

Reply:
Probably it is used so because the request is used somewhere else. `useOnce { fetchSomething() }` creates more state variables then `useRequest` **without dependencies param**